blob/ChangeLog.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+2018.12 10.3.0
```

Comment:
Please be sure to include latest commits after CR updates.

Reply:
ChangeLog is already updated.